app/models/user.rb

```diff
@@ -37,6 +37,11 @@ def self.new_with_session(params, session)
     end
   end
 
+  def self.search(input, search)
+    input.where("firstname LIKE ?", "%#{search}%")
+  end
```

Comment:
Hey Ash, is this working? It's searching the User model isn't it? So `input` = User always? Just wondering why the need to pass it as a parameter as well.